src/main/java/maksym/fedorenko/bookstore/controller/ShoppingCartController.java

```diff
@@ -60,10 +57,11 @@ public CartItemDto addCartItem(
     )
     @PutMapping("/cart-items/{cartItemId}")
     @PreAuthorize("hasRole('USER')")
-    public CartItemDto updateCartItem(
+    public CartDto updateCartItem(
```

Comment:
advice for the future: don't add changes, irrelevant to PR scope

this is a good tone, since that way PRs are simpler and easier to review

I doubt that on some real project you'll make your colleagues happy with large PRs :)

the less changes - the faster PR will be reviewed

Reply:
Sorry, I want to test for one moment, so I decided to change the return types for each endpoint to return CartDto after the interesting lection :)